app/assets/stylesheets/edit.scss

```diff
@@ -23,33 +23,3 @@
     text-align: center;
   }
 }
-
-// TODO - Tidy up during the week
```

Comment:
Is there reason for removing these styles? Can you add that you're doing this and reason to the PR description?